### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by andreysakhno**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -3,16 +3,29 @@
...
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split('\n')
+    .map(rule => rule.replace(';', '').trim())
+    .filter(rule => rule)
```
Comment (VitaliyBodnar) [inline src/convertToObject.js:20]: let's optimize your code
- you can delete this filter and add a condition to the reduce method, for instance 
```
.reduce((prev, rule) => {
 if (rule) {
  ...
 }
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/convertToObject.js [hunk 2]
```diff
@@ -3,16 +3,33 @@
...
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split('\n')
+    .reduce((prev, rule) => {
+      const next = { ...prev };
+      const currentRule = rule.replace(';', '');
```
Comment (anastasiiavorobiova) [inline src/convertToObject.js:21]: Consider using split here or replace all occurrences

**Review by anastasiiavorobiova — COMMENTED** [review]
Great work! Added a tiny comment to fix

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a tiny comment to fix

**Pull request closed without merging**